README.md

```diff
@@ -1,5 +1,4 @@
 # branch_testing
 branching demo
 
-The irreducible nature of the prime contruct are inconcievable to feeble mind. Whatevs lol. 
-ffffff
\ No newline at end of file
+second try
```

Comment:
have you thought about using 2nd instead of second?

Reply:
that is a good idea. automation of lang is desirable.